volume/volume.go

```diff
@@ -1,7 +1,9 @@
 package volume
 
-const DefaultDriverName = "local"
+// DefaultDriverName is the name for local owners of volumes.
+const DefaultDriverName string = "local"
```

Comment:
DefaultDriverName is the driver name used for the driver implemented in the `local` package.